docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,468 +1,134 @@
 ---
-title: Getting Started With Desktop Access
-description: Connect Teleport to Active Directory and log into a Windows Desktop
-videoBanner: YvMqgcq0MTQ
+title: Getting Started
+description: Automatically add Teleport Desktop Access to your infrastructure
 ---
 
-# Getting Started
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.2.6"
+>
 
-In this guide we will connect an Active Directory domain to Teleport using
-Desktop Access and log into a Windows desktop from that domain.
+The features documented on this page are available in Teleport `10.2.6` and higher.
```

Comment:
```suggestion
The features documented on this page are available in Teleport `10.2.8` and higher.
```

Reply:
Whoops, looks like I was wrong. 10.2.6 is correct. Sorry for the runaround!